### PR TITLE
*: split into two dockers etcd + pd .

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @qiuyesuifeng

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,49 +15,16 @@ docker build -t pingcap/pd .
...
-    -e ETCD_NAME=etcd0
-    -e ETCD_ADVERTISE_CLIENT_URLS=http://192.168.12.50:2379,http://192.168.12.50:4001 \
-    -e ETCD_INITIAL_ADVERTISE_PEER_URLS=http://192.168.12.50:2380 \
-    -e ETCD_INITIAL_CLUSTER_TOKEN=etcd-pd-cluster \
-    -e ETCD_INITIAL_CLUSTER=etcd0=http://192.168.12.50:2380,etcd1=http://192.168.12.51:2380,etcd2=http://192.168.12.52:2380 \
-    -e ETCD_ENDPOINTS=192.168.12.50:2379,192.168.12.51:2379,192.168.12.52:2379 \
-    -e PD_ADVERTISE_ADDR=192.168.12.50:1234 \
-    pingcap/pd
+// Start pd
+docker run -d -p 234:1234 --name pd --link etcd:etcd pingcap/pd --etcd=etcd:2379
```
Comment (qiuyesuifeng) [inline README.md]: Should be `-p 1234:1234`?

File: README.md [hunk 2]
```diff
@@ -15,49 +15,16 @@ docker build -t pingcap/pd .
...
 
 ```
-docker run -d -p 2379:2379 -p 2380:2380 -p 4001:4001 -p 1234:1234 --name pd pingcap/pd 
-```
-
-### Cluster
+// Start etcd
+export HostIP="0.0.0.0"
+docker run -d -p 2379:2379 -p 2380:2380 -p 4001:4001 --name etcd pingcap/etcd \
+    --listen-client-urls="http://0.0.0.0:2379" \
```
Comment (qiuyesuifeng) [inline README.md]: Any need to use `HostIP`?

File: docker/etcd_dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1.6
+
+MAINTAINER siddontang
+
+RUN cd /go/src && \
+    git clone --depth=1 https://github.com/coreos/etcd.git github.com/coreos/etcd && \
+    cd github.com/coreos/etcd && \
+    ./build && \
+    cp -f ./bin/etcd* /go/bin/ && \
+    rm -rf /go/src/github.com/coreos/etcd
```
Comment (qiuyesuifeng) [inline docker/etcd_dockerfile]: `cd /go/src` before remove dir?

File: README.md (hunk 2 above)
Comment (siddontang) [inline README.md]: just refer etcd docker doc.

File: docker/etcd_dockerfile (hunk 3 above)
Comment (siddontang) [inline docker/etcd_dockerfile]: no need

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @ngaut

**Comment by ngaut** [comment]
LGTM

**Pull request merged**